website/src/data/users.tsx

```diff
@@ -960,14 +944,6 @@ const Users: User[] = [
     source: 'https://github.com/orbitjs/orbit/tree/main/website',
     tags: ['opensource', 'versioning'],
   },
-  {
-    title: 'Oxidizer',
-    description: 'A Rust ORM based on tokio-postgres and refinery',
-    preview: require('./showcase/oxidizer.png'),
-    website: 'https://oxidizer.rs',
-    source: 'https://github.com/oxidizer-rs/website',
```

Comment:
✔️ Done
https://github.com/oxidizer-rs/website/issues/163

Reply:
I've updated the website link in the following PR https://github.com/facebook/docusaurus/pull/6242. Thanks, @PatelN123.